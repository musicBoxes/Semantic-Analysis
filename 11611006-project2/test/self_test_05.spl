int use_undefined_struct(){
	struct a aa;
}