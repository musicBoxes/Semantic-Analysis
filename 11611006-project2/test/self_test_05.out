Semantic Error at line 2: Struct 'a' is used without definition.
